english/polar-bear/worm_hole/worm_hole.md

```diff
@@ -0,0 +1,2 @@
+In the new dimension, millions of walruses with kaleidoscope eyes, turn and follow
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.